package.json

```diff
@@ -88,12 +88,13 @@
     "aphrodite": "1.1.0",
     "async": "^2.0.1",
     "bat-balance": "^1.0.3",
-    "bat-client": "^1.0.15",
-    "bat-publisher": "^1.0.3",
+    "bat-client": "^1.2.2",
+    "bat-publisher": "^1.2.0",
     "bignumber.js": "^4.0.4",
     "bloodhound-js": "brave/bloodhound",
     "clipboard-copy": "^1.0.0",
     "compare-versions": "^3.0.1",
+    "deepmerge": "^2.0.1",
```

Comment:
i don't see anything obviously wrong with deepmerge but fwiw we use lodash.merge in sync. i wonder if @ayumi knows why?

Reply:
no particular reason. lodash seemed well maintained.

Reply:
we can switch to loadash as well

Reply:
either way is fine